net_core/net_cmdline.cpp

```diff
@@ -87,7 +87,7 @@ NetworkCommandLine::NetworkCommandLine(wwiv::core::CommandLine& cmdline, char ne
   const auto& nws = networks_->networks();
 
   if (network_number_ < 0 || network_number_ >= size_int(nws)) {
```

Comment:
this fails for 0 networks, maybe add another if before this to say "no networks defined" or somethibg?